test/unit/services/test-get-export-csv.js

```diff
@@ -36,4 +36,20 @@ describe('services/get-export-csv', function () {
       expect(getExportCsv(orgUnit.NATIONAL.name, helper.LDU_OVERVIEW_RESULT, tabs.OVERVIEW).csv).to.include(helper.NATIONAL_OVERVIEW_HEADINGS)
     })
   })
+  describe('should format the capacity when exporting overviews', function () {
+    it('should be formatted to two decimal figures', function () {
```

Comment:
Minor: Maybe rename tests here. This should read like a sentence - outer describe, inner describe, it. So in this case:
`services/get-export-csv... should format the capacity when exporting overviews... should be formatted to two decimal figures`
should be something like 
`services/get-export-csv... should format the capacity when exporting overviews... to two decimal figures`